docs/gl_objects/projects.rst

```diff
@@ -657,3 +657,36 @@ Edit project push rules::
 Delete project push rules::
 
     pr.delete()
+
+Project protected tags
+==================
+
+Reference
+---------
+
+* v4 API:
+
+  + :class:`gitlab.v4.objects.ProjectProtectedTag`
+  + :class:`gitlab.v4.objects.ProjectProtectedTagManager`
+  + :attr:`gitlab.v4.objects.Project.protectedtags`
+
+* GitLab API: https://docs.gitlab.com/ce/api/protected_tags.html
+
+Examples
+---------
+
+Get a list of protected tags from a project::
+
+    protected_tags = project.protectedtags.list()
+
+Get a single protected tag or wildcard protected tag::
+
+    protected_tag = project.protectedtags.get('v*')
```

Comment:
This actually doesn't work for me. Just throws 404, even though all my protected tags are called v3.xx and v2.xx

Reply:
Actually it works. GitLab just implemented the protected tag feature in an unexpected way. It actually remembers the way (including wildcards how the tags where protected)